casper/src/test/scala/coop/rchain/casper/genesis/GenesisTest.scala

```diff
@@ -61,94 +61,102 @@ class GenesisTest extends FlatSpec with Matchers with BlockDagStorageFixture {
     pw.close()
   }
 
-  "Genesis.fromInputFiles" should "generate random validators when no bonds file is given" in withGenResources {
-    (
-        runtimeManager: RuntimeManager[Task],
-        genesisPath: Path,
-        log: LogStub[Task],
-        time: LogicalTime[Task]
-    ) =>
-      for {
-        _ <- fromInputFiles()(runtimeManager, genesisPath, log, time)
-        _ = log.warns.find(_.contains("bonds")) should be(None)
-      } yield log.infos.count(_.contains("Created validator")) should be(numValidators)
-  }
+  "Genesis.fromInputFiles" should "generate random validators when no bonds file is given" in taskTest(
+    withGenResources {
```

Comment:
```
Enable GenesisTest tests

They were not being run previously as `withGenResources` just returns
Task[Unit] which was not being forced to run
```
OMG we need to stop using Unit :scream:

cc @lukasz-golebiewski you'll like this :D

Reply:
IKR, I was like "I changed a bunch of behaviour in `Genesis.scala`; why the hell none of the tests failed?"